document_sftp/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====
+SFTP
+====
+
+This module allows you to access your data (= documents attached to records)
+via SFTP. There will be different virtual roots to get your browsing started,
```

Comment:
The description is not very clear. Besides you might want to terminate the sentence in this line